### PR TITLE
update ReadMe

### PR DESCRIPTION
**Pull request by karent2222**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # simpleFood
-An andoid application for recipe inspiration based on what you already have at home! 
+An andoid application for recipe inspiration based on what you already have at home! did this work?
```
Comment (chillyssa) [inline README.md]: yes it did!

**Review by chillyssa — APPROVED** [review]
looks good to me!

**Review by chillyssa — CHANGES_REQUESTED** [review]
Can you remove this?

**Review by chillyssa — APPROVED** [review]
Thanks for doing that, looks great

**Review by karent2222 — COMMENTED** [review]
good to go

**Pull request merged**